pdns/pdnsutil.cc

```diff
@@ -4407,96 +4407,255 @@ static int backendLookup(vector<string>& cmds)
+  {"backend-cmd", {true, backendCmd, GROUP_OTHER,
+   "backend-cmd BACKEND CMD [CMD..]    Perform one or more backend commands"}},
+  {"backend-lookup", {true, backendLookup, GROUP_OTHER,
+   "backend-lookup BACKEND NAME [[TYPE] CLIENT-IP-SUBNET]\n"
+   "                                   Perform a backend lookup of NAME, TYPE and CLIENT-IP-SUBNET"}},
+  {"bench-db", {true, benchDb, GROUP_OTHER,
+   "bench-db [filename]                Bench database backend with queries, one zone per line"}},
+  {"change-secondary-zone-primary", {true, changeSecondaryZonePrimary, GROUP_ZONE,
+   "change-secondary-zone-primary ZONE primary-ip [primary-ip..]\n"
+   "                                   Change secondary zone ZONE primary IP address to primary-ip"}},
+  {"check-all-zones", {true, checkAllZones, GROUP_ZONE,
+   "check-all-zones [exit-on-error]    Check all zones for correctness. Set exit-on-error to exit immediately\n"
+   "                                   after finding an error in a zone."}},
+  {"check-zone", {true, checkZone, GROUP_ZONE,
+   "check-zone ZONE                    Check a zone for correctness"}},
+  {"clear-zone", {true, clearZone, GROUP_ZONE,
+   "clear-zone ZONE                    Clear all records of a zone, but keep everything else"}},
+  {"create-bind-db", {true, createBindDb, GROUP_DNSSEC,
+   "create-bind-db FNAME               Create DNSSEC db for BIND backend (bind-dnssec-db)"}},
+  {"create-secondary-zone", {true, createSecondaryZone, GROUP_ZONE,
+   "create-secondary-zone ZONE primary-ip [primary-ip..]\n"
+   "                                   Create secondary zone ZONE with primary IP address primary-ip"}},
+  {"create-zone", {true, createZone, GROUP_ZONE,
+   "create-zone ZONE [nsname]          Create empty zone ZONE"}},
+  {"deactivate-tsig-key", {true, deactivateTSIGKey, GROUP_TSIGKEY,
+   "deactivate-tsig-key ZONE NAME {primary|secondary}\n"
+   "                                   Disable TSIG authenticated AXFR using the key NAME for ZONE"}},
+  {"deactivate-zone-key", {true, deactivateZoneKey, GROUP_ZONEKEY,
+   "deactivate-zone-key ZONE KEY-ID    Deactivate the key with key id KEY-ID in ZONE"}},
+  {"delete-rrset", {true, deleteRRSet, GROUP_RRSET,
+   "delete-rrset ZONE NAME TYPE        Delete named RRSET from zone"}},
+  {"delete-tsig-key", {true, deleteTSIGKey, GROUP_TSIGKEY,
+   "delete-tsig-key NAME               Delete TSIG key (warning! will not unmap key!)"}},
+  {"delete-zone", {true, deleteZone, GROUP_ZONE,
+   "delete-zone ZONE                   Delete the zone"}},
+  {"disable-dnssec", {true, disableDNSSEC, GROUP_DNSSEC,
+   "disable-dnssec ZONE                Deactivate all keys and unset PRESIGNED in ZONE"}},
+  {"edit-zone", {true, editZone, GROUP_ZONE,
+   "edit-zone ZONE                     Edit zone contents using $EDITOR"}},
+  {"export-zone-dnskey", {true, expotZoneDNSKey, GROUP_CDNSKEY,
+   "export-zone-dnskey ZONE KEY-ID     Export to stdout the public DNSKEY described"}},
```

Comment:
```suggestion
   "export-zone-dnskey ZONE KEY-ID     Export the public DNSKEY described to stdout"}},
```

Note that I'm not a fan of `the public DNSKEY described` -- I can't make heads/tails of it.